tests/support/server.tcl

```diff
@@ -735,8 +735,13 @@ proc start_multiple_servers {num options code} {
     uplevel 1 $code
 }
 
-proc restart_server {level wait_ready rotate_logs {reconnect 1} {shutdown sigterm}} {
+proc get_server {level} {
```

Comment:
something feels odd to me, not sure if it's the naming convention or something else.
maybe this should be in test_helper next to `proc srv`?
looks like it's not easy to change dump_server_log to take an index.
maybe we can avoid it by adding an argument to restart_server to do that?